pyGLV/examples/example_2_empty_window_with_GUI.py

```diff
@@ -11,8 +11,9 @@
 
 gGUI.init() #calls ImGUIDecorator::init()-->SDL2Window::init()
 
+
+
```

Comment:
what was the change here? seems empty lines

Reply:
Removed a comment in line 15. The new lines were mistakenly introduced.

Reply:
ok cool